controllers/api.js

```diff
@@ -24,6 +23,26 @@ const foursquare = require('node-foursquare')({
 });
 
 /**
+ * Returns a function that will wrap the given func.
+ * Instead of taking a callback, the returned function will
+ * return a promise whose fate is decided by the callback
+ * behavior of the given node function.
+ */
+function promisify(func) {
```

Comment:
I'd prefer if we could use bluebird's promisify function. See example here: https://github.com/sahat/megaboilerplate/blob/master/generators/utils.js#L9